lvsfunc.py

```diff
@@ -29,132 +56,169 @@ def compare(clip_a: vs.VideoNode, clip_b: vs.VideoNode, frames: int, mark=False,
     return sum(pairs, next(pairs))
 
 
-def stack_compare(*clips: vs.VideoNode, width=None, height=None, stack_vertical=False):
+def stack_compare(*clips: vs.VideoNode, width: int = None, height: int = None,
+                  stack_vertical: bool = False) -> vs.VideoNode:
     """
     A simple wrapper that allows you to compare two clips by stacking them.
     You can stack an infinite amount of clips.
-
+    
     Best to use when trying to match two sources frame-accurately, however by setting height to the source's
     height (or None), it can be used for comparing frames.
+    
+    Shorthand for this function is 'scomp'.
+
+    :param clips: vs.VideoNode:
+    :param width: int:  (Default value = None)
+    :param height: int:  (Default value = None)
+    :param stack_vertical: bool:  (Default value = False)
+    :returns: vs.VideoNode clip
 
-    Shorthand for this function is "scomp".
     """
     if len(set([c.format.id for c in clips])) != 1:
         raise ValueError('stack_compare: The format of every clip must be equal')
 
     if height is None:
-        height = vsutil.fallback(height, clips[0].height)
+        height = fallback(height, clips[0].height)
     if width is None:
-        width = vsutil.get_w(height, aspect_ratio=clips[0].width / clips[0].height)
+        width = get_w(height, aspect_ratio=clips[0].width / clips[0].height)
 
     clips = [c.resize.Bicubic(width, height) for c in clips]
     return core.std.StackVertical(clips) if stack_vertical else core.std.StackHorizontal(clips)
 
 
-def conditional_descale(src, height, b=1/3, c=1/3, threshold=0.003, w2x=False):
+def conditional_descale(src: vs.VideoNode, height: int, b: float = 1 / 3, c: float = 1 / 3, threshold: float = 0.003,
+                        w2x: bool = False) -> vs.VideoNode:
     """
-
-        Descales and reupscales a clip. If the difference exceeds the threshold, the frame will not be descaled.
+    Descales and reupscales a clip. If the difference exceeds the threshold, the frame will not be descaled.
```

Comment:
indentation is now inconsistent in this docstring (or at least it looks that way in GH’s diff viewer)

Reply:
That is indeed an error, fixed.